internal/cmd/operator-sdk/cli/cli.go

```diff
@@ -103,6 +104,15 @@ func GetPluginsCLIAndRoot() (*cli.CLI, *cobra.Command) {
 		manifestsv2.Plugin{},
 		scorecardv2.Plugin{},
 	)
+	// Having to call this something other than "ansible" due to it causing amibiguity issues
+	// when marking --plugins=ansible. Naming this "ansible" is likely a breaking change since
+	// our documentation mentions using --plugins=ansible instead of --plugins=ansible/v1
```

Comment:
Is this something that we are actually concerned about? As far as I am aware we have never really versioned the Operator-SDK based on the plugins so while this is likely a breaking change for users based on previous documentation is it actually a breaking change based on how we have versioned the Operator-SDK?

Reply:
Its a breaking change for  anyone looking to upgrade/migrate their existing project (or even new users, though I wouldn't call it breaking for them) as they are used to scaffolding with ansible/v1, so imo it would be nice to also have a migration guide entry to make it widely aware that from hereon we need to specify the version for ansible in the `--plugins` flag. But nothing to worry about the versioning scheme of Operator SDK. This would be in the next release. Just need to update the documentation and add highlight this addition.

Reply:
On that note, are we deprecating the previous plugin? If we are looking to remove it anytime in the near future, it would  be better if we start by adding a deprecation messages and leave it for a few SDK releases, giving folks time to migrate and make the change.

Reply:
> Its a breaking change for anyone looking to upgrade/migrate their existing project (or even new users, though I wouldn't call it breaking for them) as they are used to scaffolding with ansible/v1, so imo it would be nice to also have a migration guide entry to make it widely aware that from hereon we need to specify the version for ansible in the `--plugins` flag. But nothing to worry about the versioning scheme of Operator SDK. This would be in the next release. Just need to update the documentation and add highlight this addition.

Just to make sure I'm following correctly, the takeaway here is:
- Use `ansible/v2-alpha` as the identifier
- Update all documentation references to mention using `ansible/v1` instead of only `ansible` now (include this messaging in changelog and migration guide as well)

Does that sound correct?


> On that note, are we deprecating the previous plugin? If we are looking to remove it anytime in the near future, it would be better if we start by adding a deprecation message and leave it for a few SDK releases, giving folks time to migrate and make the change in base image.

The plan is to eventually deprecate the previous plugin but I'd prefer to wait on doing that until v2 is out of the alpha state. Once we are confident v2 is stable we can deprecate v1 for a period of time and then remove it completely.

Reply:
Updated in https://github.com/operator-framework/operator-sdk/pull/6566/commits/b8cf9779d0cf6dae23f29eb1e5289fc787b49e95